pageserver/src/tenant.rs

```diff
@@ -330,7 +330,7 @@ impl Debug for SetStoppingError {
 }
 
 struct RemoteStartupData {
-    index_part: IndexPart,
+    index_part: Arc<IndexPart>,
```

Comment:
This Arc can be avoided: https://github.com/neondatabase/neon/pull/5083

Reply:
It wasn't needed anymore afterwards? Interesting, I had stuck with that idea, but great.

Reply:
Merged it.

Reply:
Thanks!